src/common/is-node-attached.js

```diff
@@ -1,11 +1,11 @@
 // Marionette.isNodeAttached
 // -------------------------
 
-import Backbone from 'backbone';
+import domMixin from '../mixins/dom';
 
 // Determine if `el` is a child of the document
 const isNodeAttached = function(el) {
-  return Backbone.$.contains(document.documentElement, el);
+  return domMixin.containsEl(document.documentElement, el);
```

Comment:
Another point is that this $ call can be replaced by

 `return document.documentElement.contains(el)` 

So no need to abstract this function at all

According to [MDN](https://developer.mozilla.org/en-US/docs/Web/API/Node/contains), is pretty safe to use it